frame/executive/src/lib.rs

```diff
@@ -255,13 +265,25 @@ where
 	pub fn apply_extrinsic(uxt: Block::Extrinsic) -> ApplyExtrinsicResult {
```

Comment:
We should have some test that ensures that this denies an extrinsic with an invalid signature and the new one applies it.